install-dev/data/db_data.sql

```diff
@@ -0,0 +1,8 @@
+SET
+  SESSION sql_mode='';
+SET
+  NAMES 'utf8mb4';
+
+INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`)
+VALUES
+	('product_page_v2', 0, 'Experimental product page', 'Admin.Advparameters.Feature', 'This page is a work in progress. It includes new combination management features and other features under development (virtual products, packs, etc.)', 'Admin.Advparameters.Help');
```

Comment:
Why not using xml files here?

Reply:
XML files are demo fixtures, which means the merchant can choose NOT to install them.

We dont want merchant to choose whether or not this data is inserted in database 😄  . We want to be sure it is inserted all the time.

Also even if we decided to use XML files to insert this flag, it is likely we will remove the flag at 1.7.8->1.7.9 step when Product Page V2 becomes complete. So the insert would be an XML files while the removal would be SQL.

Reply:
Also using this db_data.sql is closer to the SQL upgrade scripts so the "insert Product Page V2 feature flag" logic is similar for install process and upgrade process. We want the flag to be inserted whether merchant installs or upgrades, so if we can keep the logic similar it eases future maintainance.

Reply:
XML for demo fixtures are in `install/fixtures/fashion/data/`, other data like user roles & rights, hooks, etc... are in `install/data/xml/`. I think it makes sense to use this way and to use a migration script when we'll want to remove it.

Reply:
I let you discuss this with @eternoendless he's the one who did not want XML files 😄

Reply:
For the record: fixtures use ObjectModels, and FeatureFlag is a Doctrine entity, so you can't use them.